benchmark/run.js

```diff
@@ -34,16 +34,37 @@ if (!validFormats.includes(format)) {
   return;
 }
 
+const cpuCoreSetting = cli.optional.set.find((s) => s.startsWith('CPUCORE='));
```

Comment:
Can we just create a function to get this. Code duplication makes it hard to maintain.

Reply:
sure